ffi/Cargo.toml

```diff
@@ -1,10 +1,15 @@
 [package]
 name = "mentat_ffi"
-version = "0.1.0"
+version = "0.0.1"
 authors = ["Emily Toop <etoop@mozilla.com>"]
 
+[lib]
+name = "mentat_ffi"
+crate-type = ["lib", "staticlib", "cdylib"]
+
 [dependencies]
-libc = "0.2"
+libc = "0.2.32"
```

Comment:
What's the reason for pegging to such a precise release?

Reply:
I think that was a result of my trying to find the solve the above mentioned build issue by trying to replicate the versioning in Toodle's Cargo.lock exactly and it accidentally got checked in.